spanner_orm/tests/migrations_emulator_test.py

```diff
@@ -63,6 +71,7 @@ def test_key(self):
         'string': 'string',
         'int_': 42,
         'float_': 4.2,
+        'bytes_': base64.b64encode(b'hello'),
```

Comment:
Sorry for not bringing this up earlier, but ideally base64 encodings would not need to be provided manually. Can this be handled automatically?

Reply:
Isn't that handled [here](https://github.com/google/python-spanner-orm/blob/2ae8e42841dd092efc70fa5d84324067bdd3e473/spanner_orm/condition.py#L221-L234)?

I thought this test just involves the raw database for the migration tests. In the other tests this is not needed. Please let me know.

Reply:
Thanks for clarifying.
I added an update to validate base64 encodings for an initial version. Although more strict, it will be backward compatible with future changes that can support arbitrary byte encodings.

Reply:
As discussed, could you add a comment saying it is sometimes impossible to tell if something is base64 encoded without more metadata?